surch/organization.py

```diff
@@ -37,45 +37,39 @@ def __init__(
             git_user,
             git_password,
             print_result=False,
-            organization_flag=True,
+            is_organization=True,
             repos_to_skip=None,
             repos_to_check=None,
             consolidate_log=False,
-            cloned_repos_path=constants.CLONED_REPOS_PATH,
+            cloned_repos_path=None,
             results_dir=constants.RESULTS_PATH,
             verbose=False,
             remove_cloned_dir=False,
             **kwargs):
-        """Surch instance define var from CLI or config file
-        """
         self.print_result = print_result
         self.search_list = search_list
         self.organization = organization
         self.results_dir = results_dir
         if repos_to_skip and repos_to_check:
-            lgr.warn("Can't run surch with list of exclude and includes repo ")
+            lgr.warn('You can not both include and exclude repositories.')
             sys.exit(1)
         self.repos_to_skip = repos_to_skip or []
         self.repos_to_check = repos_to_check or []
         if not git_user or not git_password:
             lgr.warn(
                 'Choosing not to provide GitHub credentials limits '
                 'requests to GitHub to 60/h. This might affect cloning.')
-            self.auth = False
+            self.creds = False
         else:
-            self.auth = True
-            self.git_user = git_user
-            self.git_password = git_password
+            self.creds = (self.git_user, self.git_password)
```

Comment:
self.creds is treated as a boolean type on line 62 and here it is treated as a tuple this may be confusing to anyone reading your code.

my suggestion is to set self.creds=None at the beginning and only set it to self.creds=(self.git_user, self.git_password) if the command-line flags are given.